projects/plugins/jetpack/tests/e2e/specs/plugin-updater.test.js

```diff
@@ -1,29 +1,25 @@
-import { doClassicConnection } from 'jetpack-e2e-commons/flows';
+import { test as baseTest, expect } from '../fixtures/base-test.js';
+import { doClassicConnection } from 'jetpack-e2e-commons/flows/index.js';
 import {
 	execShellCommand,
 	execSyncShellCommand,
 	execWpCommand,
 	prepareUpdaterTest,
-} from 'jetpack-e2e-commons/helpers/utils-helper';
+	resolveSiteUrl,
+} from 'jetpack-e2e-commons/helpers/utils-helper.cjs';
 import {
 	Sidebar,
 	PluginsPage,
 	DashboardPage,
 	JetpackPage,
-} from 'jetpack-e2e-commons/pages/wp-admin';
-import { testStep } from 'jetpack-e2e-commons/reporters';
-import { prerequisitesBuilder } from 'jetpack-e2e-commons/env';
+} from 'jetpack-e2e-commons/pages/wp-admin/index.js';
+import { prerequisitesBuilder } from 'jetpack-e2e-commons/env/index.js';
 
-/**
- *
- * @group pre-connection
- * @group update
- */
-describe( 'Jetpack updater', () => {
-	beforeAll( async () => {
+const test = baseTest.extend( {
```

Comment:
Why do we extend the baseTest here, instead of using `beforeAll` hook? It is not available in playwright-test?